src/libraries/System.Text.Json/gen/JsonSourceGenerator.Emitter.cs

```diff
@@ -1099,7 +1114,14 @@ private static SourceText GetRootJsonContextImplementation(ContextGenerationSpec
 
                 GetLogicForDefaultSerializerOptionsInit(contextSpec.GeneratedOptionsSpec, writer);
 
-                writer.WriteLine();
+                writer.WriteLine($"""
+
```

Comment:
Use raw literals like this will end up baking in the newline used in this file, regardless of where the source generator ends up running. Since we're already using them, seems like we're ok with that?

Reply:
The `SourceWriter` class splits multi-line strings and inserts the necessary indentation before each line:

https://github.com/dotnet/runtime/blob/ed7aef86caf0505456a203009d3bdd4b7290b7d8/src/libraries/Common/src/SourceGenerators/SourceWriter.cs#L41-L60